version

```diff
@@ -46,9 +47,17 @@ done
 
 # Infer version
 # Number of commits since branch point
+IS_DIRECT_CHILD=$( git log origin/master..8228cea)
```

Comment:
Shouldn't this reference the $BRANCH_POINT instead of a hardcoded commit hash? 
```suggestion
IS_DIRECT_CHILD=$( git log origin/master..$BRANCH_POINT)
```